examples/overview-react-native/src/AliceViews.js

```diff
@@ -0,0 +1,158 @@
+import React from 'react';
+import { Text, TextInput, View } from 'react-native';
+import styles from './Styles';
+import Button from './Button';
+import Clipboard from '@react-native-clipboard/clipboard';
+
+const sleep = (milliseconds) => new Promise(resolve => setTimeout(resolve, milliseconds));
```

Comment:
What do you and Jay think of using TypeScript with React? It could be helpful long-term, especially in avoiding errors.

Reply:
These are demo programs and we don't want people to think they need to use TypeScript to use Reach. We use TS internally for all our libraries, but we don't want to impose it on people.

Reply:
I agree.